sdk/core/azure_core_amqp/Cargo.toml

```diff
@@ -37,7 +37,7 @@ typespec_macros.workspace = true
 tracing-subscriber = { workspace = true, features = ["env-filter"] }
 
 [features]
-default = ["fe2o3_amqp"]
+default = ["fe2o3_amqp", "fe2o3-amqp/native-tls"]
```

Comment:
Do you need to support alternatives? If so, you'll need a separate `fe2o3_amqp_native_tls` feature that probably depends on `fe2o3_amqp`; though, I'm not sure how congruent that is with `reqwest` where that's also used to just provide a blanket `HttpClient` implementation for `reqwest::Client`.

Reply:
I don't need to support alternatives, others might. So they can use the same no-default-features option that we do with azure core. 

They'll have to add the fe2o3_amqp feature to get things to work (since we don't support alternatives), but they can manually configure fe2o3 however they want.